payments-core/src/main/java/com/stripe/android/payments/bankaccount/navigation/CollectBankAccountResult.kt

```diff
@@ -26,5 +27,6 @@ sealed class CollectBankAccountResult : Parcelable {
 
 @Parcelize
 data class CollectBankAccountResponse(
-    val intent: StripeIntent
+    val intent: StripeIntent,
+    val linkAccountSession: LinkAccountSession
```

Comment:
Do we now want to expose the session object then?

Reply:
That is what iOS is doing, just doing the same thing to be consistent. The thing that we really need is `PaymentAccount`

Reply:
👍 let's do this then.